packages/cli/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to the Zowe CLI package will be documented in this file.
 
 ## Recent Changes
 
+- Enhancement: Add the commands `zowe daemon enable` and `zowe daemon disable`. These commands enable end-users to set up daemon mode without having to download a separate executable and place it by hand into some directory.
+- **Next Breaking**: The Daemon-related class named `Processor` was renamed to `DaemonDecider`.
+
```

Comment:
Should this be modified to follow the format of:

Bugfixes
Enhancements
Breaking Changes

Reply:
I assume that you are talking about the order of the items?